packages/fe-fpm-writer/test/integration/__snapshots__/index.test.ts.snap

```diff
@@ -420,6 +441,15 @@ Object {
                     }
                   }
                 }
+              },
+              \\"views\\": {
```

Comment:
as expected, the snapshot only shows an example with exactly one view entry. Adding a custom view to a list of existing views is not supported.

Reply:
Done

Reply:
ok